pkg/app/sysconf/config.go

```diff
@@ -152,8 +156,24 @@ const (
 	PolicyCheck PolicyMode = "check" // Check requests against the policy if the request has a token or client cert.
 )
 
+// Health configures the health check system.
+type Health struct {
+	// DBPath is the location of the SQLite database file used to store health check results.
+	// Relative paths are relative to DataDir.
+	// Defaults to "health/checks.sqlite3".
+	DBPath string    `json:"dbPath,omitempty"`
```

Comment:
Devil's advocate: does the path need to be configurable at all? 
- What scenarios would you need to change the path?
- Do we really need to support those scenarios?

If not, I suggest removing this configuration and fixing the location within the data dir.